mesonbuild/modules/pkgconfig.py

```diff
@@ -368,7 +372,16 @@ def generate(self, state, args, kwargs):
         elif len(args) > 1:
             raise mesonlib.MesonException('Too many positional arguments passed to Pkgconfig_gen.')
 
-        subdirs = mesonlib.stringlistify(kwargs.get('subdirs', ['.']))
+        dataonly = kwargs.get('dataonly', False)
+        if dataonly:
+            default_subdirs = []
+            if len(kwargs.get('libraries'), []) +
+               len(kwargs.get('libraries_private'), []) +
```

Comment:
__[Flake8]__ [E111] indentation is not a multiple of four ([link](https://sider.review/gh/repos/19784232/pulls/5775#issue-4435333))

<sub>posted by Sider</sub>